typescript/infra/config/warp.ts

```diff
@@ -1,3 +1,4 @@
+import { getRegistry } from '@hyperlane-xyz/cli';
```

Comment:
why is this command in the cli and not the sdk itself if we're reusing it across diff places

not a blocker for this PR but wondering for future

Reply:
Good point yes. Should this even be in the SDK, but registry package, no?

Reply:
sdk shouldn't use FileSystem because the UI uses it.